diff -uNr bc-1.07.1.orig/bc/fix-libmath_h bc-1.07.1/bc/fix-libmath_h
--- bc-1.07.1.orig/bc/fix-libmath_h	2017-04-07 17:20:02.000000000 -0500
+++ bc-1.07.1/bc/fix-libmath_h	2019-10-16 09:13:14.260105974 -0500
@@ -1,9 +1,9 @@
-ed libmath.h <<EOS-EOS
-1,1s/^/{"/
-1,\$s/\$/",/
-2,\$s/^/"/
-\$,\$d
-\$,\$s/,\$/,0}/
-w
-q
-EOS-EOS
+#! /bin/bash
+sed -e '1   s/^/{"/' \
+    -e     's/$/",/' \
+    -e '2,$ s/^/"/'  \
+    -e   '$ d'       \
+    -i libmath.h
+
+sed -e '$ s/$/0}/' \
+    -i libmath.h
